define HAVE_STDINT_H for DEBUG build

From: MURAOKA Taro <koron.kaoriya@gmail.com>

Problem: HAVE_STDINT_H is defined for no debug build on for MSWIN.
It cause problem when compiling debug versoin with if_python3 feature.

Solution: move the definition block of HAVE_STDINT_H to out of NODEBUG
block.

diff --git a/src/Make_mvc.mak b/src/Make_mvc.mak
index 29d8b0eb4..f5def80ab 100644
--- a/src/Make_mvc.mak
+++ b/src/Make_mvc.mak
@@ -626,6 +626,11 @@ CFLAGS = $(CFLAGS) /MP
 !endif
 !endif
 
+# VC10 or later has stdint.h.
+!if $(MSVC_MAJOR) >= 10
+CFLAGS = $(CFLAGS) -DHAVE_STDINT_H
+!endif
+
 
 !ifdef NODEBUG
 VIM = vim
@@ -649,11 +654,6 @@ OPTFLAG = $(OPTFLAG) /GL
 CFLAGS=$(CFLAGS) $(WP64CHECK)
 !endif
 
-# VC10 or later has stdint.h.
-!if $(MSVC_MAJOR) >= 10
-CFLAGS = $(CFLAGS) -DHAVE_STDINT_H
-!endif
-
 # Static code analysis generally available starting with VS2012 (VC11) or
 # Windows SDK 7.1 (VC10)
 !if ("$(ANALYZE)" == "yes") && ($(MSVC_MAJOR) >= 10)
